config/routes.rb

```diff
@@ -1,6 +1,19 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  
+  # MY CODE
+  # get '/users', to: "users#index"
+  # get '/users/:id', to: "users#show"
+  # get '/users/:user_id/posts', to: "posts#index"
+  # get '/users/:user_id/posts/:id', to: "posts#show"
+  
+  #CODE REVIEWER optional comment
+  # [OPTIONAL] While this is going to work fine, I will encourage you to take advantage of resources by having something like:
```

Comment:
- Hey, great job adding the routes. please, kindly remove these commented pieces of code to make your codebase clean and maintainable.